tests/conftest.py

```diff
@@ -17,9 +17,9 @@ def pytest_configure(config):
 
 
 @pytest.fixture(scope='session')
-def filesdir(tmpdir_factory):
-    LocalPath = tmpdir_factory.getbasetemp().__class__  # noqa: N806
-    return LocalPath(__file__).new(basename='')
+def filesdir(tmp_path_factory):
+    LocalPath = tmp_path_factory.getbasetemp().__class__  # noqa: N806
```

Comment:
Do we still need this (instead of returning a `pathlib.Path` instance directly)?

Reply:
I am not quite sure why we need that `filesdir` actually. Do you mean we could do without a factory?